test/test_container_docker/ubuntu_20_04_ros2_multi/bashrc

```diff
@@ -0,0 +1,103 @@
+# ~/.bashrc: executed by bash(1) for non-login shells.
```

Comment:
we should add "main" folder for this files and the same files should e only sym-linked for specific versions